code/index.html

```diff
@@ -1,30 +1,190 @@
+          <input 
+            type="email" 
+            name="mailAddress" 
+            id="mailAddress" 
+            class="textfield"
+            placeholder="yourmail@mail.se"
+            required
+          >
+        </div>
+
+        <div class="form-group">
+          <label class="label-left" for="arrival">I'll arrvive on</label>
+          <input 
+          type="date" 
+          name="arrival"
+          id= "arrival"
+          value ="2021-07-16"
+          min="2021-07-16"
+          max="2021-07-18"
+          >
+        </div>
+
+        <!--Two inputs with choices, one checkbox, one radio-->
+        <div class="form-choose">
+          <label class="label-left" for="bring1">I can bring</label>
+            <div class="bringbox">
+              <input 
+                type="checkbox" 
+                name="bring" 
+                id="bring1" 
+                value="tent"
+              >
+              <label for="bring1" class="label-right">tent</label>
+              
+              <input 
+                type="checkbox" 
+                name="bring" 
+                id="bring2" 
+                value="hayfork"
+              >
+              <label for="bring2" class="label-right">hay fork</label>
+              
+              <input 
+                type="checkbox" 
+                name="bring" 
+                id="bring3" 
+                value="scythe"
+              >
+              <label for="bring3" class="label-right">scythe</label>
+
+              <input 
+                type="checkbox" 
+                name="bring" 
+                id="bring4" 
+                value="guitar"
+              >
+              <label for="bring4" class="label-right">guitar</label>
+            </div>
+        </div>
+
+        <div class="form-choose">
+          <label class="label-left" for="member">I'm a member *</label>
+            <div class="memberbox">
+              <input 
+                type="radio" 
+                id="member" 
+                name="member" 
+                value="yes"
+                required
+              >
+              <label for="member" class="label-right">Yes</label>
+
+              <input 
+                type="radio" 
+                name="member" 
+                id="noMember" 
+                value="no"
+              >
+              <label for="noMember" class="label-right">No</label>
+            </div>
+        </div>
+        <button class="form-submit" type="submit">Join the harvest!</button>
+      </form>
     </section>
-    <!-- dont forget to link the javascript file here! -->
-  </body>
-</html>
+
+    <section class="accordion-container">
+      <section class="accordion">
+        <!-- Accordion -->
+        <h2>Good to know</h2>
+
+        <div id="question1" class="question"><span class="icon"> </span>Is the event suitable for children?</div>
+        <article id="answer1" class="answer">
+          <p>Lorem ipsum...</p>
+        </article>
```

Comment:
Please check the indentation, it will help you as a developer for the future and makes the code easier to read 😋